internal/expr/query.go

```diff
@@ -100,6 +102,20 @@ func (pe *PreparedExpr) Query(args ...any) (ce *QueryExpr, err error) {
 	return &QueryExpr{outputs: pe.outputs, sql: pe.sql, args: qargs}, nil
 }
 
+type NullT struct {
+	Value reflect.Value
+	Valid bool
+}
+
+func (nt *NullT) Scan(src any) error {
+	if src == nil {
+		nt.Valid = false
+		return nil
+	}
+	nt.Valid = true
```

Comment:
is it an issue that `nt.Valid` can be `true` when `ConvertAssign` returns an error?